src/plugins/inputs/join.js

```diff
@@ -2,8 +2,11 @@ module.exports = function(network, chan, cmd, args) {
 	if (cmd !== "join") {
 		return;
```

Comment:
It does if for any reason we add code checking `== false`, as `undefined` evaluates to `false` but is not `false`. My main issue is that this doesn't keep consistency in returned types, but I won't strongly oppose since this code will evolve anyway.